crates/oxc_cli/src/command.rs

```diff
@@ -44,6 +44,12 @@ impl Command {
                 .action(ArgAction::Append)
                 .help("This option allows you to specify patterns of files to ignore (in addition to those in .eslintignore).")
             )
+            .arg(
+                Arg::new("max-warnings")
+                  .long("max-warnings")
+                  .required(false)
+                  .help("This option allows you to specify a warning threshold, which can be used to force oxc_lint to exit with an error status if there are too many warning-level rule violations in your project.")
+              )
```

Comment:
We still need a test, it's at the bottom.

Reply:
Done.